cmd/monaco/integrationtest/v2/test-resources/integration-all-configs/project/automation/config.yaml

```diff
@@ -7,6 +7,12 @@ configs:
     name: '[CA] Notify Capability about new Jira Issues'
     template: jira-issue-workflow.json
     parameters:
+      comment: "// optional import of sdk modules"
+      jiraJs:
+        type: file
+        path: "jiraJs.txt"
```

Comment:
nit: why not.... `jira.js`? 😅

Reply:
haha yeah, intellij treats it as a javascript file and complains with parsing error because of our templating :P